src/v3/src/util/formUtils.ts

```diff
@@ -189,10 +186,6 @@ const getConfigIdpButtonRemediations = (
           name: idpConfig.text,
         };
         const redirectUri = `${baseUrl}/sso/idps/${idpConfig.id}?stateToken=${stateHandle}`;
-        if (idpConfig.className) {
```

Comment:
Same with this re: classname deprecation.

Reply:
Ticket for tracking: https://oktainc.atlassian.net/browse/OKTA-699701